<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgWatercolorOptions</class>
 <widget class="QWidget" name="WdgWatercolorOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>328</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>255</horstretch>
    <verstretch>255</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>290</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="labelGravity">
     <property name="text">
      <string>Gravity vector:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="gravityLayout">
     <item>
      <widget class="QLabel" name="labelX">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KisDoubleSliderSpinBox" name="gravityX" native="true"/>
     </item>
     <item>
      <widget class="QLabel" name="labelY">
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KisDoubleSliderSpinBox" name="gravityY" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelBrush">
     <property name="text">
      <string>Brush type:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="brushType">
     <item>
      <property name="text">
       <string>Simple</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Wet-on-dry</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Wet-on-wet</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Blobby</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Crunchy</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisDoubleSliderSpinBox</class>
   <extends>QWidget</extends>
   <header>kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
